eng/scripts/Language-Settings.ps1

```diff
@@ -121,7 +121,7 @@ function Get-AllPackageInfoFromRepo($serviceDirectory)
     if ($ciProps) {
       # CheckAOTCompat is opt _in_, so we should default to false if not specified
       $shouldAot = GetValueSafelyFrom-Yaml $ciProps.ParsedYml @("extends", "parameters", "CheckAOTCompat")
-      if ($shouldAot) {
+      if ($null -ne $shouldAot) {
```

Comment:
This is not actually necessary, as if it wasn't _enabled_ we were falling to setting it to `disabled`. Because that's the default, that's actually what we want, but I'd rather the logic be consistent with `BuildSnippets`.